### ISSUE TITLE
[FEATURE] Mass tracking enabled by default

### ISSUE THREAD

**Issue by dnolen** [exodus4d/pathfinder#313]
Currently pilots are reporting that mass tracking is not occurring correctly with in the connections logged in pathfinder. 

There is an option to toggle "Save Mass" that has to be enabled by default but we need a clear jump log of all hole traffic to track mass for hole longevity and knowing accurately how much mass is left on the hole when taking large fleets out or calculating rolling mass.

**Comment by dkrotil** [comment]
There is no mass tracking, it is NOT possible in CREST. Pledge here https://forums.eveonline.com/default.aspx?g=posts&t=490255&find=unread for adding ship info to location endpoint.

**Comment by exodus4d** [comment]
Someone from *CCP* told ship information is available somewhere in the new **Fleet** Endpoint. I haven´t checked this.... So we could think about adding **Fleet** - **CREST** support to *Pathfinder*...

But this will grant access to lots of  "sensitive data" for *Pathfinder*. Not sure if people would like that.

**Comment by dkrotil** [comment]
This means, that if the character is member of corporation fleet it will show in CREST ship information otherwise not ?

**Comment by exodus4d** [comment]
Something like this... I think. Haven´t checked the Endpoint yet. But being in a Fleet is probably a requirement to get it out of the Endpoint.... Not very sexy. And a lot more CREST calls need to be done to get that information ....

**Comment by InfiniZ** [comment]
Correct, shipType endpoint is only available currently in a fleet, making this much harder to track in Pathfinder.

**Comment by exodus4d** [comment]
@InfiniZ https://forums.eveonline.com/default.aspx?g=posts&m=6639828#post6639828

**Comment by vivace-io** [comment]
Referencing new ESI support: https://esi.tech.ccp.is/dev/#!/Live/get_characters_character_id_ship

Should be able to calculate an estimate from that, could go a step further and match the `ship_item_id` with assets from the XML API to get the exact fit and be more accurate.

**Comment by exodus4d** [comment]
### New Feature for "_mass tracking_":

- Ship mass will be tracked for wormhole jumps
- New "_Connections_" panel was added to the UI
- Multiple connections can be selected and compared by its mass status 
  1. `click` at connection to select
  2. `ctrl`+ `click` adds more connections to that that panel
  3. new context menu option "_select connections_" added to systems
- Connections that have **signature mapped to** will show *total mass* 
- Your **current ship mass** can be added to mass calculation
- Warnings will be shown in case your current ship:
  - Does not fit through that type of wormhole
  - Your ship mass will make the wormhole collapse after jump
- Estimation for "_mass left_" is available (in percent)

![select_system_connections](https://user-images.githubusercontent.com/9299033/33849102-e742d416-deaf-11e7-9651-6b5b6e974565.gif)
